packages/block-editor/src/autocompleters/style.scss

```diff
@@ -4,6 +4,10 @@
 	.block-editor-block-icon {
 		margin-right: $grid-unit-10;
 	}
+
+	&.is-selected .block-editor-block-icon {
+		color: inherit !important;
```

Comment:
Was there a way to choose between coloured or block and white icons? Should we always choose black and white for autocomplete maybe?

Reply:
Even `!important`'s can be overridden by plugins if they want it enough — at least until we can shadow-DOM these components :) 

I'd tend to agree that color for blocks should be enforced in this view, it's a way to enforce contrast.

Reply:
> Was there a way to choose between coloured or block and white icons? Should we always choose black and white for autocomplete maybe?

If we choose white/black, I think all we need to do is delete `showColors` [here](https://github.com/WordPress/gutenberg/blob/3d8b192c3df3b02f4c1efc9d8154b715dc189600/packages/block-editor/src/autocompleters/block.js#L103).

![image](https://github.com/WordPress/gutenberg/assets/54422211/4fb43f7a-b463-466e-a453-f5df0cc376e8)

In my experience, some block plugins color their icons to highlight their branding. Therefore, I think it is better to respect the colored icon when the block is not selected.